legofy/legofy_gui.py

```diff
@@ -0,0 +1,89 @@
+import legofy
+import tkinter as tk
+import tkinter.ttk as ttk
+from tkinter import filedialog
+import tkinter.messagebox as tkmsg
+
+LEGO_PALETTE = ('none', 'solid', 'transparent', 'effects', 'mono', 'all', )
+
+class LegofyGui(tk.Tk):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.wm_title("Legofy!")
+        self.iconbitmap('./assets/brick.ico')
```

Comment:
This doesn't work when invoked from any other directory, than the one containing the script.